packages/util-dev/src/TestLogger.ts

```diff
@@ -129,8 +129,9 @@ export const createLogger = (options: TestLoggerOptions = {}) => {
   };
 
   const logger = Object.fromEntries((<LoggerEntry[]>Object.entries(logLevels)).map((_) => [_[0], getLogFunction(_)]));
+  const reset = () => <void>(<unknown>(messages.length = 0));
```

Comment:
Adding a new method requires unit test coverage, but wouldn't it be better to just create a new instance for each test?

Reply:
Test added with 0557f8148

> wouldn't it be better to just create a new instance for each test?

As general rule, yes, I agree.
Considering our tests, we usually need to pass the logger to several dependencies of the `class`/`function` to test: doing that would bean to destroy several object at the end of each test and re-creating them at each new test; this would make our tests much more complex.